src/rigid.jl

```diff
@@ -186,9 +186,9 @@ function qd_rigid(fixed, moving, mxshift::VecLike, mxrot::Union{Number,VecLike};
     #TODO make sure isrotation(initial_tfm.linear) returns true, else throw a warning and an option to terminate? do I still allow the main block to run?
     if initial_tfm == IdentityTransformation() || isrotation(initial_tfm.linear)
     else
-        @show "WARNING: initial_tfm is not a rigid transformation"
+        @warn "initial_tfm is not a rigid transformation"
```

Comment:
This caused problems during tests. I'll revert and address this in a separate PR.